atom/common/node_bindings.cc

```diff
@@ -119,9 +124,25 @@ NodeBindings::~NodeBindings() {
   uv_sem_destroy(&embed_sem_);
   uv_close(reinterpret_cast<uv_handle_t*>(&dummy_uv_handle_), nullptr);
 
-  // Destroy loop.
-  if (uv_loop_ != uv_default_loop())
-    uv_loop_delete(uv_loop_);
+  // Clean up worker loop
+  if (uv_loop_ == &worker_loop_) {
```

Comment:
Will it be clearer if all the code under the `if` statement is moved to a separate method?

Reply:
Good idea. Committed 132d9705f6cd4907aeb2f9721851e697c1cf8a4c